src/Void/Init.fs

```diff
@@ -35,11 +35,13 @@ module Init =
             Channel [

```

Comment:
Haha, I know. I think I inadvertently became a microservices guy. Did you look at the `CommandBar` service? I love the way that played out in the specs---felt like I was hitting a perfect testing level, testing a sane amount of complexity and the tests very unlikely to be brittle.